samples/Mvc.Client/Startup.cs

```diff
@@ -49,6 +49,11 @@ public class Startup {
                 options.ClientSecret = "98f1bf028608901e9df91d64ee61536fe562064b";
             });
 
+            app.UseLinkedInAuthentication(options => {
```

Comment:
OK, when I get all the other issues fixed, I can commit with Client id and secret